packages/tslint-config/test/rules/blueprint-html-components/true/test.tsx.lint

```diff
@@ -1,13 +1,12 @@
+import { H3, Pre } from "@blueprintjs/core";
+
+<H3>Subtitle</H3>
+<Pre>block</Pre>
```

Comment:
I was hoping to test the replacement of `pre` -> `Pre` as well. can you change this line to `pre`?